### PR TITLE
W9_ExtraTips

### PR DESCRIPTION
**Pull request by filipzeman**
- Add extra-tips component (tpl, style), 

- add requested svgs to src/gfx/icons/,
 
- update health-index.nunj with include extra-tips component,

- remove .container from default.nunj

- add .container to welcome.nunj

### CODE REVIEW

File: src/styles/components/extra-tips.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  }
+
+  &__element-content {
+    @include media-breakpoint-up(lg) {
+      margin-top: 40px;
+    }
+  }
+
+  &__icon {
+    @include size(45px,45px);
```
Comment (fidelman) [inline src/styles/components/extra-tips.scss]: why 2 arguments?

File: src/styles/components/extra-tips.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    color: map-get($theme-colors, primary);
+
+    @include media-breakpoint-up(lg) {
+      margin-left: 0;
+    }
+  }
+
+  &__text {
+    margin-top: 10px;
+    font-size: 13px;
```
Comment (fidelman) [inline src/styles/components/extra-tips.scss]: create vars for font-sizes

File: src/styles/components/extra-tips.scss [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    font-weight: 300;
+    line-height: 20px;
```
Comment (fidelman) [inline src/styles/components/extra-tips.scss]: better to keep in var

File: src/tpl/components/extra-tips.nunj [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+                        activate facial muscles which send the
+                        proper stimulus to your brain and your
+                        spirit will lift straight away.
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="row extra-tip">
+            <div class="col-lg-10">
+                <div class="extra-tip__element-aside">
```
Comment (fidelman) [inline src/tpl/components/extra-tips.nunj]: what about to call it aside, instead of element-aside?

**Review by fidelman — CHANGES_REQUESTED** [review]

File: src/styles/components/extra-tips.scss (hunk 1 above)
Comment (filipzeman) [reply src/styles/components/extra-tips.scss]: Fixed (forgot that $height has $width as default value )

**Review by fidelman — APPROVED** [review]

**Pull request merged**